zipkin-lens/src/components/App/Layout.test.jsx

```diff
@@ -42,8 +42,37 @@ describe('<Layout />', () => {
         },
       },
     );
-    const helpLink = getByTitle('Support');
+    const helpLink = queryByTitle('Support');
```

Comment:
It might be a deficiency of the assertions framework but I found `getByTitle` to give a better error message when the element isn't found.

Reply:
thanks will change in the next PR if it works. just noticed things seemed arbitrarily different

Reply:
74 |       },
      75 |     );
    > 76 |     expect(getByTitle('Dependencies Page')).not.toBeInTheDocument();
         |            ^
      77 |   });
      78 | });
      79 | 

      at Object.getElementError (node_modules/@testing-library/dom/dist/config.js:34:12)
      at args (node_modules/@testing-library/dom/dist/query-helpers.js:71:38)
      at getByTitle (node_modules/@testing-library/dom/dist/query-helpers.js:54:17)
      at Object.it (src/components/App/Layout.test.jsx:76:12)

Reply:
so we can either use them differently and comment why, or use queryByTitle consistently and not comment anything, or do some error handling?

Reply:
I'll leave this one to you. I don't really like unintuitive bombs in code, where we do one thing and then secret knowledge why we have to do it differently somewhere else.

Reply:
Agree it's not intuitive and I don't like it, will try to think of a good way, may just a helper method we define, or a monkey patch. For reference, the big difference is `getBy` prints out a dump of the DOM if an element isn't found while `queryBy.toBeInDocument` just says something like `undefined not in document`. Though it does seem like a bug in `toBeInDocument` to me.

Reply:
yeah I didn't mean to shoot the messenger either, I know javascript in particular is rife with tribal knowledge as a way of doing things.. something I've never acclimated to